great_expectations/core/usage_statistics/usage_statistics_record_schema.json

```diff
@@ -1,5539 +0,0 @@
-{
-  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
We can probably remove our jsonschema dependency right? Anything else to get rid off after these changes?